extensions/blocks/logo-gallery/editor.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * Internal dependencies
+ */
+import registerJetpackBlock from '../../utils/register-jetpack-block';
```

Comment:
Note that we've moved files a bit around, utils are now under `/extensions/shared`. Please rebase. :-)

https://github.com/Automattic/jetpack/pull/11971